wormchain/Makefile

```diff
@@ -78,3 +93,50 @@ bootstrap:
 clean:
 	rm -rf build/wormchaind build/wormchaind-* build/**/*.db build/**/*.wal vue
 	echo "{\"height\":\"0\",\"round\":0,\"step\":0}" > build/data/priv_validator_state.json
+
+get-heighliner:
+	git clone https://github.com/strangelove-ventures/heighliner.git
+	cd heighliner && go install
```

Comment:
We may want to make this:

```
get-heighliner:
        go install github.com/strangelove-ventures/heighliner@latest
```

to avoid the git clone

Reply:
Very good point! I was actually supposed to remove heighliner completely from the Make file. Looks like I only took out 1 of 2 targets. I'll remove this completely now.